app/views/contexts/list_context.py

```diff
@@ -46,8 +48,15 @@ def _build_list_items_context(
         edit_block_id,
         remove_block_id,
         primary_person_edit_block_id,
+        filter_list_item_ids,
```

Comment:
Not totally against it but I personally don't think `filter` is clear without looking at code, I can't tell if it is `filter in` or `filter out`. I don't really have a great suggestion but could be like `for_list_items_ids`?

Reply:
Fair point. I will change it to `for_list_items_ids` as that is also consistent with the `for_list` parameter.